Tasks.php

```diff
@@ -344,7 +344,11 @@ private static function getNohupCommandIfPresent()
         if (SettingsServer::isWindows()) {
             return '';
         }
-
-        return 'nohup';
+        
+        if (!empty(@shell_exec('nohup --version'))) {
```

Comment:
```suggestion
$requiredExecutablesCheck = StaticContainer::get(RequiredExecutablesCheck::class);

        if ($requiredExecutablesCheck->isNohupPresent()) {
            return 'nohup';
        } else {
            return '';
        }
```

You need to add this use statement too

`use Piwik\Plugins\GoogleAnalyticsImporter\Diagnostic\RequiredExecutablesCheck;`